public/app/features/dashboard/components/ShareModal/ShareModal.tsx

```diff
@@ -122,16 +123,34 @@ export class ShareModal extends React.Component<Props, State> {
   }
 
   render() {
+    const styles = getStyles();
     const { dashboard, panel } = this.props;
     const activeTabModel = this.getActiveTab();
     const ActiveTab = activeTabModel.component;
 
     return (
-      <Modal isOpen={true} title={this.renderTitle()} onDismiss={this.props.onDismiss}>
+      <Modal
+        isOpen={true}
+        title={this.renderTitle()}
+        onDismiss={this.props.onDismiss}
+        className={styles.container}
+        contentClassName={styles.content}
+      >
         <TabContent>
           <ActiveTab dashboard={dashboard} panel={panel} onDismiss={this.props.onDismiss} />
         </TabContent>
       </Modal>
     );
   }
 }
+
+const getStyles = () => {
+  return {
+    container: css({
+      paddingTop: '8px',
+    }),
+    content: css({
+      padding: '24px 16px 16px 16px',
```

Comment:
we should use GrafanaTheme2 for spacing here

Reply:
yes, just needed to decide what to do with the class - https://raintank-corp.slack.com/archives/CHKLEJ668/p1689328145961219

all is fixed now! 🙏 please review it again when you have time